lollipop/types.py

```diff
@@ -434,15 +434,15 @@ def dump(self, value, *args, **kwargs):
         result = []
         for idx, (item_type, item) in enumerate(zip(self.item_types, value)):
             try:
-                result.add(item_type.dump(item, *args, **kwargs))
+                result.append(item_type.dump(item, *args, **kwargs))
             except ValidationError as ve:
                 errors_builder.add_errors({idx: ve.messages})
         errors_builder.raise_errors()
 
         return super(Tuple, self).dump(result, *args, **kwargs)
 
     def __repr__(self):
-        return '<{klass} of {item_types}>'.format(
+        return '<{klass} of {item_type}>'.format(
```

Comment:
Looks like there is a bug again: you reference value with "item_type" (which is incorrect since it's multiple types) and then you pass the value as "item_types"